device/iot-device-samples/send-event/src/main/java/samples/com/microsoft/azure/sdk/iot/SendEvent.java

```diff
@@ -239,4 +205,4 @@ else if (protocolStr.equals("mqtt_ws"))
 
         System.out.println("Shutting down...");
     }
-}
+}
```

Comment:
nit: file should end on a new line I think.

Reply:
Correct

Reply:
> will do.